WPMedia/fusion-news-theme-blocks
# Tmedia 178 add copyright to article body

## Description
Adding Copyright info at the bottom of the body if it's included in the globalContent object

## Jira Ticket
- [TMEDIA-178](https://arcpublishing.atlassian.net/browse/TMEDIA-178)

## Acceptance Criteria
- If the copyright field is present for a story then it should display below the article body
- It should be left-aligned, in the same styling as paragraph text
- Tests and documentation are updated as necessary

## Test Steps
1. `git checkout TMEDIA-187-add-copyright-to-article-body`
2. Run local: Fusion-News-Theme % `npx fusion start -f -l @wpmedia/article-body-block`
3. Add/edit meta tab copyright in Composer article (I used https://corecomponents.arcpublishing.com/composer/edit/MXCQIAVGQJHTLBLSG7NZXORS3Q/)
- Verify it is updated in test page: http://localhost/pagebuilder/editor/curate?p=pNi98SIA93PpVgts&v=vgwTKAkei3PpVgts

## Effect Of Changes
### Before
Copyright would not show at end of body

### After
Copyright should show at end of body
![image](https://user-images.githubusercontent.com/2287238/116137363-7357a000-a6a1-11eb-88e1-c662631ad001.png)

## Dependencies or Side Effects
N/A

## Author Checklist
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
